content/millennium/dstu2/scheduling/appointment.md

```diff
@@ -55,3 +55,86 @@ List an individual Appointment by its id:
 
 <%= headers status: 200 %>
 <%= json(:dstu2_appointment_read) %>
+
+### Errors
+
+The common [errors] may be returned.
+
+
+## Create
+
+Create a new Appointment.
+
+    POST /Appointment
+
+_Implementation Notes_
+
+* [Appointment.status] must be set to `proposed`.
```

Comment:
Maybe these should use the tick marks similar to needs-action (they're fields in the response) rather than brackets? Also no need to call out that it's part of the Appointment resource.